sda-commons-client-oidc/src/main/java/org/sdase/commons/client/oidc/rest/model/TokenResource.java

```diff
@@ -0,0 +1,45 @@
+package org.sdase.commons.client.oidc.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * An access token resource according to <a
+ * href="https://tools.ietf.org/html/rfc6749#section-5.1">RFC 6749 section 5.1</a>
+ */
+public class TokenResource {
+  @JsonProperty("token_type")
+  private String tokenType;
+
+  @JsonProperty("access_token")
+  private String accessToken;
+
+  @JsonProperty("expires_in")
```

Comment:
I thought this value is "exp"? Can that be changed? 
https://tools.ietf.org/html/rfc7519#section-4.1.4

Reply:
This is not a token but the response that _delivers_ the token. It’s more convenient to use this expiration value in seconds than to use with the Unix-timestamp in the token where you need to make sure you don’t confuse the different time zones.

Reply:
I see - thanks for the explanation!